src/utilities.cpp

```diff
@@ -20,27 +20,21 @@ limitations under the License.
 #include <iomanip>
 #include <limits>
 #include <set>
+#include <vector>
+#include <iostream>
 #include <sstream>
-#include <stdexcept>
 
 namespace libcellml {
 
-bool convertToDouble(const std::string &candidate, double *value)
+double convertToDouble(const std::string &candidate)
```

Comment:
Why has the signature for `convertToDouble()` been changed? If anything, I would just rewrite it slightly to be `bool convertToDouble(const std::string &candidate, double &value)`.